libraries/Microsoft.Bot.StreamingExtensions/RequestHandler.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Bot.StreamingExtensions
+{
+    /// <summary>
+    /// Implemented by classes used to process incoming requests sent over an <see cref="IStreamingTransport"/> and adhering to the Bot Framework Protocol v3 with Streaming Extensions.
+    /// </summary>
+    public abstract class RequestHandler
+    {
```

Comment:
I'm confused again. :( 

Why is this abstract?

Reply:
Because it has an abstract method in it for how the consumer wants to process and dispatch the incoming requests.